### PR TITLE
project documentation

### PR DESCRIPTION
**Pull request by arshiii16**
**Pull Request Overview**
This pull request adds the documentation of my project

**TODO or Help Wanted**
Please let me know if there is anything that needs changes

**Build**
 Ran npm build.

**Author**
Signed-off-by: Balutoiu Ana-Maria [balutoiu.ana.maria2003@gmail.com]

### CODE REVIEW

File: docs/project/2024/ana_maria.balutoiu/index.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+| [50k Potentiometer (RV1)](https://www.optimusdigital.ro/en/potentiometers/1885-50k-mono-potentiometer.html?search_query=50k&results=72) | Adjusts the alarm frequency. | [1.49 RON](https://www.optimusdigital.ro/en/potentiometers/1885-50k-mono-potentiometer.html?search_query=50k&results=72) |
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [rppal](https://github.com/golemparts/rppal) | Raspberry Pi Peripheral Access Library | Provides an interface for accessing various Raspberry Pi peripherals |
+| [panic-halt](https://github.com/korken89/panic-halt) | panic handler that halts the program | Used to handle panics gracefully |
+| [elf2uf2](https://github.com/JoNil/elf2uf2-rs) | for flashing onto microcontrollers | Converts Rust executables (in ELF format) to UF2 format |
```
Comment (alexandruradovici) [inline docs/project/2024/ana_maria.balutoiu/index.md:98]: This is not a library.

File: docs/project/2024/ana_maria.balutoiu/index.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+## Motivation
+
+I chose this project because it is straightforward and doesn’t require complex components or intricate circuitry. The Hall Effect sensor, 555 timer IC, and a few other basic components are affordable and readily available. Also, I think everybody wants to know when someone opens the doors around the house, whether it’s for security reasons or just to keep track of household activity. This alarm serves that purpose efficiently. Finally, it’s a great way to learn about basic electronics, sensors, and timers.
+
+## Architecture 
+
+![Architecture](Architecture.png)
+Here’s a description of the system components and their interactions:
+
+1) Hall Effect Sensor:
```
Comment (alexandruradovici) [inline docs/project/2024/ana_maria.balutoiu/index.md:26]: Is this part rendered corectly?

File: docs/project/2024/ana_maria.balutoiu/index.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+### Week 7 - 19 May
+
+### Week 20 - 26 May
+
+## Hardware
+
+The magnetic door alarm project uses the following hardware components:
+
+1) Hall Effect Sensor (3144): Detects the presence of a magnet (door open).
```
Comment (alexandruradovici) [inline docs/project/2024/ana_maria.balutoiu/index.md:64]: Is this part rendered corectly? Please try it locally using `npm start`.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
Please fill the pull request description following the template.

File: docs/project/2024/ana_maria.balutoiu/index.md (hunk 2 above)
Comment (arshiii16) [reply docs/project/2024/ana_maria.balutoiu/index.md:26]: Yes!

File: docs/project/2024/ana_maria.balutoiu/index.md (hunk 3 above)
Comment (arshiii16) [reply docs/project/2024/ana_maria.balutoiu/index.md:64]: I think it's rendered corectly

File: docs/project/2024/ana_maria.balutoiu/index.md (hunk 1 above)
Comment (arshiii16) [reply docs/project/2024/ana_maria.balutoiu/index.md]: Thank you! I deleted the elf2uf2 tool from the list

**Review by arshiii16 — COMMENTED** [review]
I updated the description and tested if everything is rendered corectly

**Review by alexandruradovici — APPROVED** [review]

**Review by irina-nita — APPROVED** [review]

**Pull request merged**